sfepy/discrete/common/extmods/_geommech.pyx

```diff
@@ -7,8 +7,7 @@ cimport cython
 cimport sfepy.discrete.common.extmods._geommech as gmch
 from types cimport int32
 
-from sfepy.discrete.common.extmods._fmfield cimport\
-    (FMField, FMF_SetCell, array2fmfield4)
+from sfepy.discrete.common.extmods._fmfield cimport (FMField, FMF_SetCell, array2fmfield4)
```

Comment:
Due to an unhadled issue in skbuild, https://github.com/scikit-build/scikit-build/issues/302

Reply:
Would using
```
from sfepy.discrete.common.extmods._fmfield cimport (
    FMField, FMF_SetCell, array2fmfield4,
)
```
also help? I would avoid the line longer than 79 chars, but if it is not possible, so be it :)